tests/detect-flowbits/test.rules

```diff
@@ -0,0 +1,7 @@
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb1; sid:1;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb2; sid:2;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb3; sid:3;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb3|fb4; sid:4;)
+alert ip any any -> any any (msg:"Flowbit isnotset ored flowbits"; flowbits:isnotset, fb1 | fb2 ; sid:5;)
+alert ip any any -> any any (msg:"Flowbit unset"; flowbits:unset,myflow2; sid:6;)
+alert ip any any -> any any (msg:"Flowbit unset"; flowbits:toggle,myflow2; sid:7;)
```

Comment:
The result of this rule varies based on what was the previous state of the flowbits variable, so I think we'll have to decide first on what we want here, so we can settle on the test checks and on the rule message...
(acc to https://suricata.readthedocs.io/en/latest/rules/flow-keywords.html?#flowbits)